.github/workflows/ci.yaml

```diff
@@ -363,53 +363,6 @@ jobs:
           git submodule update --init --recursive --jobs 7
           ./project/scripts/sbt "community-build/testOnly dotty.communitybuild.CommunityBuildTestC"
 
-      - name: Show dependency tracking file
```

Comment:
Did you remove this on purpose? Or is it another git revert issue

Reply:
Definitely not intentional.